auditlogs.go

```diff
@@ -106,17 +109,16 @@ func (a AuditLogFilter) String() string {
 //
 // API Reference: https://api.cloudflare.com/#audit-logs-list-organization-audit-logs
 func (api *API) GetOrganizationAuditLogs(organizationID string, a AuditLogFilter) (AuditLogResponse, error) {
-	uri := "/organizations/" + organizationID + "/audit_logs" + fmt.Sprintf("%s", a)
-
-	res, err := api.makeRequest("GET", uri, nil)
-	if err != nil {
-		return AuditLogResponse{}, err
+	uri := url.URL{
+		Path:       path.Join("/organizations", organizationID, "audit_logs"),
```

Comment:
Organization endpoints are deprecated and `/organization` should be replaced with `/accounts` instead. We should get the API documentation updated to reflect this a well.

Reply:
Offf, good to know.  I'll switch to the preferred API.  How can I get audit logs via `/accounts`?  It looks like I'm missing something in the API.

Regardless, until then, this at least fixes something that was abjectly broken.

Reply:
the `s/organization/account` swap is a 1:1 mapping from my understanding. Same structure, payloads, etc. give it a whirl and confirm it works as other parts of the API have been updated to reflect this (example https://api.cloudflare.com/#organizations-properties)

i'm not certain on this one but if it works, we can get in touch with the API team to update the documentation.

Reply:
I have no idea how I was supposed to know that, but you're right!, it works!  PR updated.

Thank you!

Reply:
unfortunately it is a little buried but that is what code review is for 😄